### ISSUE TITLE
`kill_my_queries` not killing my queries

### ISSUE THREAD

**Issue by matkoniecz** [drolbr/Overpass-API#416]
I can enter http://overpass-api.de/api/kill_my_queries multiple times. Every time I get blank page*, http://overpass-api.de/api/status still lists running querries

*in the past I remember getting some text that queries were killed. entire content of blank page is: ```<html><head><link rel="alternate stylesheet" type="text/css" href="resource://gre-resources/plaintext.css" title="Wrap Long Lines"></head><body><pre></pre></body></html>```

see #214 for the same bug type (fixed, closed)

**Comment by mmd-osm** [comment]
Cannot reproduce. Just tested the steps mentioned below and it works as expected.



1. Start long running query in turbo
2. Check /api/status -> query active
3. Kill my queries -> result:

``` 
Killing query (pid 12345) from IP abc ...
Done!
Killing query (pid 12345) from IP abc ...
Done!
```
4. Check /api/stauts -> query gone
5. Call kill_my_query again -> empy result.

**Comment by matkoniecz** [comment]
Now I tried again and this time query died and I got info that my query was killed (curiously, this time I got "Killing query (pid 19947) from IP <IP> ... Done!" three times - with single query and every time pid was the same).

> Note that you cannot influence 1 slots available now. Slot available after: 2017-07-28T10:49:35Z, in 4 seconds. by killing queries. That's part of rate limiting and by design.

Yeah, I understand that "Kill my queries" is not "kill my rate limiting".

> Are you behind some load balancer or proxy with changing IP addresses maybe?

Nothing that I setup, my IP is quite stable, though maybe ISP did something weird.

> Also, the query in turbo immediately shows an error popup.

In my case I was running query from program downloading data, not from overpass turbo.

I will close and reopen if I will be able to reproduce again (I encountered problem with unkillable query before and I was pretty sure that I caught it this time - next time if I encounter it I will make sure to post also /status ouput).
